src/bin/cql-stress-cassandra-stress/settings/option/schema.rs

```diff
@@ -37,51 +34,53 @@ impl SchemaOption {
     pub fn print_settings(&self) {
         println!("Schema:");
         println!("  Keyspace: {}", self.keyspace);
-        println!("  Replication Factor: {}", self.replication_factor);
-        println!("  Replication Strategy: {}", self.replication_strategy);
```

Comment:
The previous prints in cql-stress where inconsistent with the way c-s print them.
See output from (java) c-s:
```
Schema:
  Keyspace: keyspace1
  Replication Strategy: org.apache.cassandra.locator.SimpleStrategy
  Replication Strategy Options: {replication_factor=1}
  Table Compression: null
  Table Compaction Strategy: null
  Table Compaction Strategy Options: {}
```

Reply:
Not sure if you meant to imply that they are consistent now, but it looks like they are still inconsistent, i.e. there is no `Replication Strategy:` and `Table Compaction Strategy:` lines. 

I don't have an issue with this personally unless something depends on this format (SCT? You probably know it better than I do because you were running it recently).

Reply:
You are right, sorry for the confusion.

Speaking of SCT, I just double-checked and ran simple SCT test-cases on the branch from this PR. The tests pass, no runtime errors. I also grep'd SCT project to search for possible usages of this format - didn't find anything. So I think we can assume that SCT doesn't depend on this format.